recipe/meta.yaml

```diff
@@ -1,4 +1,4 @@
-{% set version = "2018.12.28" %}
+{% set version = "2019.01.06" %}
```

Comment:
Odd that this is in the future. I'll merge tomorrow if their are no objections

Reply:
oh, we can revert this. This was a late night fail.

Reply:
probably fine